ios/test_app.rb

```diff
@@ -201,10 +214,15 @@ def make_project!(xcodeproj, project_root, target_platform)
   destination = File.join(nearest_node_modules(project_root), '.generated', target_platform.to_s)
   dst_xcodeproj = File.join(destination, xcodeproj)
 
-  # Copy/link Xcode project files
-  FileUtils.mkdir_p(dst_xcodeproj)
-  FileUtils.cp(File.join(src_xcodeproj, 'project.pbxproj'), dst_xcodeproj)
-  FileUtils.ln_sf(File.join(src_xcodeproj, 'xcshareddata'), dst_xcodeproj)
+  # Copy Xcode project files
+  FileUtils.mkdir_p(destination)
+  FileUtils.cp_r(src_xcodeproj, destination)
+  name, _display_name = app_name(project_root)
+  unless name.nil?
+    xcschemes_path = File.join(dst_xcodeproj, 'xcshareddata', 'xcschemes')
+    FileUtils.cp(File.join(xcschemes_path, 'ReactTestApp.xcscheme'),
+                 File.join(xcschemes_path, "#{name}.xcscheme"))
```

Comment:
@Saadnajmi: Would this be enough? It just creates a copy of the scheme. Doesn't change the installed app name or anything. We could address that in a separate PR if desired.

Reply:
@anandrajeswaran since he's the one who suggested it

Does that mean that there is 2 identical schemes now? ReactTestApp and <app name> ?

Reply:
SHouldn't it just create one scheme that is the {name} of the app?

Reply:
I'm contemplating whether to keep `ReactTestApp` to be backwards compatible. It doesn't hurt, I think. And we can remove it in a later version.

Reply:
That's fine by me to not have breaking changes. Thanks for actually implementing the feature request!